node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
```

Comment:
Warn if no value is returned from array functions (`map`, `filter` etc)
https://eslint.org/docs/rules/array-callback-return
```js
// Old
const arr = array.map(a => {
  a.isBroken = true
})
// New
const arr = array.map(a => {
  a.isBroken = false
  return a
})
```
*Not auto-fixable but is only a warning and is usually a very bad mistake*

Reply:
woohoo! no side-effects!

Reply:
Is this going to be applied to every array function? What about `forEach`? Would a whitelist be required for [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)?

Reply:
forEach is excluded (see the eslint docs https://eslint.org/docs/rules/array-callback-return#rule-details)